### PR TITLE
add powertools repo for gpgme-devel

### LINKED ISSUE #7996: Trouble building from recent centos stream 8
Instructions are not clear how to proceed with gpgme on centos.

I installed the dependencies but I am getting:

```
[kehannon@scos-crio-build cri-o]$ make
go build  -trimpath  -ldflags '-s -w -X github.com/cri-o/cri-o/internal/version.buildDate='2024-04-10T15:18:15Z' ' -tags "containers_image_ostree_stub  exclude_graphdriver_btrfs btrfs_noversion exclude_graphdriver_devicemapper libdm_no_deferred_remove  seccomp selinux " -o bin/crio github.com/cri-o/cri-o/cmd/crio
go build github.com/proglottis/gpgme:
# pkg-config --cflags  -- gpgme
Package gpgme was not found in the pkg-config search path.
Perhaps you should add the directory containing `gpgme.pc'
to the PKG_CONFIG_PATH environment variable
Package 'gpgme', required by 'virtual:world', not found
pkg-config: exit status 1
make: *** [Makefile:185: bin/crio] Error 1
```

### PR DESCRIPTION
**Pull request by kannon92**
#### What type of PR is this?
/kind documentation

#### What this PR does / why we need it:
I was trying to building cri-o on centos and I found some instructions that didn't work.
#### Which issue(s) this PR fixes:

Fixes #7996 

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Review by sohankunkerkar — APPROVED** [review]
/lgtm

File: install.md [hunk 1]
```diff
@@ -129,6 +129,8 @@ yum install -y \
 - `CentOS 8` (or higher): `pkgconfig` package is replaced by `pkgconf-pkg-config`
 - By default btrfs is not enabled. To add the btrfs support, install the
   following package: `btrfs-progs-devel`
+- `CentOS 8` (or higher): `gpgme-devel` can be installed with the powertools repo.
+  (`yum install -y gpgme-devel --enablerepo=powertools`)
```
Comment (kwilczynski) [inline install.md:133]:
```suggestion
- `CentOS 8` (or higher): `gpgme-devel` can be installed using the **powertools** repository.
  (`yum install -y gpgme-devel --enablerepo=powertools`)
```

Comment (kannon92) [reply install.md]: Patched

**Comment by kannon92** [comment]
> You need to fix : https://github.com/cri-o/cri-o/actions/runs/8646854444/job/23707059803?pr=8000

Ah! thanks. Pushed a fix.

File: install.md [hunk 2]
```diff
@@ -129,6 +129,9 @@ yum install -y \
 - `CentOS 8` (or higher): `pkgconfig` package is replaced by `pkgconf-pkg-config`
 - By default btrfs is not enabled. To add the btrfs support, install the
   following package: `btrfs-progs-devel`
+- `CentOS 8` (or higher): `gpgme-devel` can be
+  installed with the powertools repo.
+  (`yum install -y gpgme-devel --enablerepo=powertools`)
```
[the remark this answers is not in the document]
Comment (kannon92) [reply install.md:134]: I'll test on a centos 9 VM and I'll update if that works.

[the remark this answers is not in the document]
Comment (kannon92) [reply install.md:134]: you are correct!

[the remark this answers is not in the document]
Comment (kannon92) [reply install.md:134]: pushed up a fix for it.

**Review by saschagrunert — APPROVED** [review]

**Pull request merged**